gui/app.pro

```diff
@@ -9,7 +9,6 @@ CONFIG(debug, debug|release) {
 # For Mac, x86 and x64, but not PPC binary.   Ignored on other OSes.
```

Comment:
Let's drop this hunk while we're here. PPC has been dead for a long time.

Reply:
ok